lumicks/pylake/tests/test_channels/test_channels.py

```diff
@@ -981,3 +982,23 @@ def start(self):
 
     with pytest.raises(TypeError, match=invalid_range):
         s[BadType(0)]
+
+
+def test_low_level_construction():
+    start = 1388534400000000000
+    with pytest.raises(ValueError, match="Starting timestamp must be larger than"):
+        _ = make_continuous_slice(np.array([1, 2, 3]), start - 1, int(1e9 / 78125))
```

Comment:
Is the `_` necessary?
```suggestion
        make_continuous_slice(np.array([1, 2, 3]), start - 1, int(1e9 / 78125))
```

Reply:
Not needed indeed.